airflow/www_rbac/views.py

```diff
@@ -69,6 +69,8 @@
 from sqlalchemy.orm import joinedload
 from wtforms import SelectField, validators
 
+from mlflow.tracking.artifact_utils import _download_artifact_from_uri
```

Comment:
We want to use our version of mlflow right?

Reply:
We can if we want to. Isn't a necessity, the import function is pretty standard. Even if we do use our own version, this import will remain the same -- only the mlflow pip installation would change.